### ISSUE TITLE
River fails to build for recent Alpine edge on CI

### ISSUE THREAD

**Issue by ifreund** [riverwm/river#860]
Likely this is due to a recent update of musl on alpine edge, needs further investigation.

**Comment by novakne** [comment]
I think this is about LFS64 changes in musl

https://git.musl-libc.org/cgit/musl/commit/?id=246f1c811448f37a44b41cd8df8d0ef9736d95f4

See the release note for musl 1.2.4 https://musl.libc.org/releases.html

Some discussion https://www.openwall.com/lists/musl/2022/09/30/3

See the alpine log https://git.alpinelinux.org/aports/log/main/musl/APKBUILD

Could be already fixed in zig master though I [see](https://github.com/ziglang/zig/pull/16098)

**Comment by ifreund** [comment]
Indeed, this seems like upstream zig issue https://github.com/ziglang/zig/issues/15610.

It would be good to test that this works on zig master so that we can confidently wait for Zig 0.11 to fix this issue.
